controllers/userController.ts

```diff
@@ -21,11 +21,12 @@ export async function registerUser(
   username: string,
   email: string,
   password: string,
-  profile: typeof Image,
+  profile: string,
```

Comment:
this type should match the comment above. there's 3 possible types here that you have to choose from.

1. typeof Image
2. string
3. Optional<Image>

Since the controller's purpose is to model the backend endpoints, the types should represent as closely as possible what the backend need. Think about that and take a look at the backend to decide.